common/source/docs/common-rtf.rst

```diff
@@ -64,9 +64,11 @@ VTOL/QuadPlanes from Partners
 * `FlyDragonDroneTech - FDG30 <https://www.droneassemble.com/product/vtol-uav-6-hours-endurance-with-1kg-payload-for-survey-serveillance/>`__
 * `FlyDragonDroneTech - FDG50F <https://www.droneassemble.com/product/hybrid-vtol-uav-7-hours-endurance-with-10kgs-payload/>`__
 * :ref:`Holybro Swan-K1 <common-Swan-K1>`
+* :ref:`MakeFLyEasy - Fighter Hand Throw <common-makeflyeasy-fighter-hand-throw>`
 * :ref:`MakeFLyEasy - Fighter VTOL <common-makeflyeasy-fighter-vtol>`
 * `MakeFLyEasy - Freeman 2300 <https://www.aliexpress.com/item/10000223137957.html?spm=a2g0o.store_home.productList_1076398524.pic_3>`__
 * `MakeFLyEasy - Freeman 2100 <https://www.aliexpress.com/item/10000223137957.html?spm=a2g0o.store_home.productList_1076398524.pic_2>`__
```

Comment:
https://www.uavmodel.com/products/makeflyeasy-striver-mini-4-1-4-2-2100mm-vtol-uav?variant=41656322654362

Reply:
where is this model linked now?